hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-lt.tex

```diff
@@ -1,38 +1,22 @@
-% This file is part of hyph-utf8 package and resulted from
-% semi-manual conversions of hyphenation patterns into UTF-8
-% in October 2008 by Mojca & Arthur.
-%
-% Source available on:
-% - http://www.vtex.lt/tex/littex/index.html
-%
-% Authors:
-%
-% First version in LT encoding (modified T1):
-% - Vytas Statulevičius <vytas at vtex.lt>
-% - Yannis Haralambous
-%   (Vilnius, March 4, 1992)
-% Conversion into Latin 7 and aditional support files (babel, fonts):
-% - Sigitas Tolušis <sigitas at vtex.lt>
-%   (2002-11-20)
-% Adaptation for hyph-utf8:
-% - Mojca & Arthur (see below), October 2008
-%
-% The copyright statement of this file is:
-%
-%    Do with this file whatever needs to be done in future for the sake of
-%    "a better world" as long as you respect the copyright of original file.
-%
-% If you want to change this file, rather than uploading directly to CTAN,
-% we would be grateful if you could send it to us (http://tug.org/tex-hyphen)
-% or ask for credentials for SVN repository and commit it yourself;
-% we will then upload the whole "package" to CTAN.
-%
-% For more unformation see
-%
-%    http://tug.org/tex-hyphen
-%
-%------------------------------------------------------------------------------
+% title: Hyphenation patterns for Lithuanian
+% copyright: Copyright (c) 1992 Sigitas Tolušis and Yannis Haralambous, Vilnius, March 4, 1992
+% notice: This file is part of the hyph-utf8 package.
+%     See http://www.hyphenation.org for more information
+%     Source available on http://www.vtex.lt/tex/littex/index.html
+% language:
+%     name: Lithuanian
+%     tag: lt
+% licence: >
+%     Do with this file whatever needs to be done in future for the sake of
+%     “a better world” as long as you respect the copyright of original file.
```

Comment:
I wonder where the origin of this sentence is (I fear it's some wording from our project). Should we clarify with upstream?

Reply:
Yes, we should clarify upstream.